docs/hotfix.md

```diff
@@ -0,0 +1,68 @@
+# How to make a hotfix release
+
+This document describes how to generate a hotfix release for backplane. 
```

Comment:
I think it would be better to focus on the backplane-cli here. And if needed, we can add the backplane-api process in the backplane-api repo.
Then we can create/update the backplane doc in source to link to both.